app/src/main/java/org/simple/clinic/drugs/PrescriptionRepository.kt

```diff
@@ -61,7 +61,8 @@ class PrescriptionRepository @Inject constructor(
 
   fun softDeletePrescription(prescriptionUuid: UUID): Completable {
     return Completable.fromAction {
-      dao.softDelete(prescriptionUuid, deleted = true)
+      dao.softDelete(prescriptionUuid, deleted = true, updatedAt = Instant.now())
+      dao.updateSyncStatus(listOf(prescriptionUuid), SyncStatus.PENDING)
```

Comment:
I think we should run these two calls inside a transaction. Thoughts?

Reply:
Good point. I'll make the change.